make.js

```diff
@@ -515,17 +519,21 @@ function populateSDKFeatureGroups(apis) {
     }
 }
 
-function setPrerequisiteCalls() {
+function setPrerequisiteAndCleanupCalls() {

```

Comment:
>setPrerequisiteAndCleanupCalls [](http://example.com/codeflow?start=9&length=30)

Any reason for this additional step?  Could you just consume prereqsAndCleanupMap directly in the test template files?

Reply:
I decided to keep this part outside the individual file gen since it sets up lists of the actual api calls instead of the names. In the event that any prerequisite or cleanup calls need to come from outside a the test's feature group (which may actually end up happening with a couple of Groups APIs), this'll be a cleaner way to get those call objects over passing the entire SDKFeatureGroups map.